src/Program.cs

```diff
@@ -109,6 +47,21 @@ private static void SetupParser(ParserSettings settings)
 
         private static int New(NewOptions opts)
         {
+            Version version = opts.Version ?? VersionManager.GetLatestDarkRiftVersion();
+
+            // Executes the command to download the version if it doesn't exist
+            if (Pull(new PullOptions()
+            {
+                Version = version,
+                Pro = opts.Pro,
+                Platform = opts.Platform,
+                Force = false
+            }) != 0)
+            {
+                Console.Error.WriteLine(Output.Red("An error occured while trying to download the version required, exiting New"));
```

Comment:
Can we trim the 'exiting New' part of this? I don't think it would make sense to a user that doesn't know the CLI internals

Reply:
That's actually not my code, i didn't touch that part but sure :D